vsphere, baremetal, platform none vmxnet3 bz (#31965)
openshift/openshift-docs: modules/installation-requirements-user-infra.adoc

The bootstrap and control plane machines must use {op-system-first} as the operating system.

ifdef::bare-metal,agnostic[]
[IMPORTANT]
====

> If the `platform: none` field is defined in the `install-config.yaml` file, virtual machines (VMs) configured to use virtual hardware version 14 or greater might result in a failed installation. It is recommended to configure VMs with virtual hardware version 13. This is a known issue that is being addressed in link:https://bugzilla.redhat.com/show_bug.cgi?id=1935539[BZ#1935539].

Reviewer: Introducing the bug link like this is more common in the collection:

```suggestion
If the `platform: none` field is defined in the `install-config.yaml` file, virtual machines (VMs) configured to use virtual hardware version 14 or greater might result in a failed installation. It is recommended to configure VMs with virtual hardware version 13. For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1935539[BZ#1935539].
```
